res/Sectors/M1105/M1105.xml

```diff
@@ -2880,14 +2880,14 @@ Left:   0041 0040 0039 0038 0037 0036 0035 0034 0033 0032 0031 0030 0029 0028 00
     <Y>5</Y>
     <X>4</X>
     <Name>Darret</Name>
-    <DataFile Source="The Core Route Project" Type="SecondSurvey" Author="Bari Z. Stafford Sr." Ref="https://web.archive.org/web/20030406042613/http://www.geocities.com/traveller_core_route/proj004/proj004.html">darret.txt</DataFile>
+    <DataFile Source="General Turokan's Expedition to the Rim" Type="SecondSurvey" Author="Bari Z. Stafford Sr." Ref="https://web.archive.org/web/20030406042613/http://www.geocities.com/traveller_core_route/proj004/proj004.html">darret.txt</DataFile>
```

Comment:
For future reference, when the file type changes these need a `Type="SecondSurvey"` or `Type="TabDelimited`" attribute added.